libmozevent/phabricator.py

```diff
@@ -91,8 +91,9 @@ def update_state(self, build):
             return
 
         # Check this build has been awaited between tries
+        exp_backoff = (2 ** (self.retries - retries_left + 1) - 1) * self.sleep
```

Comment:
`self.retries - retries_left` won't work, as self.retries is a dict, no?

Reply:
Weird that the unit test on code review did not pick that up

Reply:
I'm adding a local unit test to trigger it